aws/redshift/test_redshift_security_group_does_not_allow_all_ips_access.py

```diff
@@ -10,6 +10,7 @@
 
 
 @pytest.mark.redshift
+@pytest.mark.xskip(reason="Redshift security groups are only supported on EC2-Classic accounts")
```

Comment:
the more I know :)

I wonder if we could detect this somehow

Reply:
It does return a very specific error response. I'll open an issue.

Reply:
👍

I was wondering if the could detect that at the account level. Should we put EC2-Classic tests in a separate folder?

Reply:
Oh I see. From what I saw, it wasn't really exposed by the API besides in error responses, but worth further investigation.

Reply:
We could also delete the test and make an issue to add it back if there's demand for testing EC2-Classic accounts. 

Dunno, skipping is good for now.